src/types.ts

```diff
@@ -26,12 +28,12 @@ export type WaterfallItems = Array<{
     };
 }>;
 
-type WaterfallInterval = {
+export type WaterfallInterval = {
     name: string;
     color: string;
     type: 'block' | 'line';
-    start: string; // timing name
-    end: string; // timing name
+    start: number; // timing name
```

Comment:
As far as I remember, this is a string value, because instead of setting a number value here, you should set the name of the waterfall item. And the interval will be used as a template in this case.

Reply:
I will recheck this, not sure why I changed this

Reply:
will probably do it today in the evening after work

Reply:
I have updated the PR